server/src/utils/instrumentation.ts

```diff
@@ -19,11 +19,13 @@ export const hostMetrics =
 export const apiMetrics =
   process.env.IMMICH_API_METRICS == null ? metricsEnabled : process.env.IMMICH_API_METRICS === 'true';
 export const repoMetrics =
-  process.env.IMMICH_IO_METRICS == null ? metricsEnabled : process.env.IMMICH_IO_METRICS === 'true';
+  process.env.IMMICH_IO_METRICS == null ? metricsEnabled : process.env.IMMICH_IO_METRICS === 'true'; // check usage of repoMetrics. If it is IO related why not call it IO?
```

Comment:
The wording was repo everywhere at first, then I changed the env to IO since repo doesn't mean anything to an immich user without context. You can change the variable here and `MetricRepository` to also use `io` for consistency.

Reply:
Done! I just commented that there so I didn't forget to ask you guys.